drivers/l3gd20/l3gd20.py

```diff
@@ -0,0 +1,209 @@
+L3GD20_I_AM_L3GD20_TR                = const(0xD5) 
+L3GD20_MODE_POWERDOWN                = const(0x00) 
+L3GD20_MODE_ACTIVE                   = const(0x08) 
+L3GD20_OUTPUT_DATARATE_1             = const(0x00) 
+L3GD20_OUTPUT_DATARATE_2             = const(0x40) 
+L3GD20_OUTPUT_DATARATE_3             = const(0x80) 
+L3GD20_OUTPUT_DATARATE_4             = const(0xC0) 
+L3GD20_X_ENABLE                      = const(0x02) 
+L3GD20_Y_ENABLE                      = const(0x01) 
+L3GD20_Z_ENABLE                      = const(0x04) 
+L3GD20_AXES_ENABLE                   = const(0x07) 
+L3GD20_AXES_DISABLE                  = const(0x00) 
+L3GD20_BANDWIDTH_1                   = const(0x00) 
+L3GD20_BANDWIDTH_2                   = const(0x10) 
+L3GD20_BANDWIDTH_3                   = const(0x20) 
+L3GD20_BANDWIDTH_4                   = const(0x30) 
+L3GD20_FULLSCALE_250                 = const(0x00) 
+L3GD20_FULLSCALE_500                 = const(0x10) 
+L3GD20_FULLSCALE_2000                = const(0x20)  
+L3GD20_FULLSCALE_SELECTION           = const(0x30) 
+L3GD20_SENSITIVITY_250DPS            = 8.75        #!< gyroscope sensitivity with 250 dps full scale [DPS/LSB]  
+L3GD20_SENSITIVITY_500DPS            = 17.50       #!< gyroscope sensitivity with 500 dps full scale [DPS/LSB]  
+L3GD20_SENSITIVITY_2000DPS           = 70.00       #!< gyroscope sensitivity with 2000 dps full scale [DPS/LSB] 
+L3GD20_BlockDataUpdate_Continous     = const(0x00) 
+L3GD20_BlockDataUpdate_Single        = const(0x80) 
+L3GD20_BLE_LSB                       = const(0x00) 
+L3GD20_BLE_MSB                       = const(0x40) 
+L3GD20_HIGHPASSFILTER_DISABLE        = const(0x00) 
+L3GD20_HIGHPASSFILTER_ENABLE         = const(0x10) 
+L3GD20_INT1                          = const(0x00) 
+L3GD20_INT2                          = const(0x01) 
+L3GD20_INT1INTERRUPT_DISABLE         = const(0x00) 
+L3GD20_INT1INTERRUPT_ENABLE          = const(0x80) 
+L3GD20_INT2INTERRUPT_DISABLE         = const(0x00) 
+L3GD20_INT2INTERRUPT_ENABLE          = const(0x08) 
+L3GD20_INT1INTERRUPT_LOW_EDGE        = const(0x20) 
+L3GD20_INT1INTERRUPT_HIGH_EDGE       = const(0x00) 
+L3GD20_BOOT_NORMALMODE               = const(0x00) 
+L3GD20_BOOT_REBOOTMEMORY             = const(0x80) 
+L3GD20_HPM_NORMAL_MODE_RES           = const(0x00) 
+L3GD20_HPM_REF_SIGNAL                = const(0x10) 
+L3GD20_HPM_NORMAL_MODE               = const(0x20) 
+L3GD20_HPM_AUTORESET_INT             = const(0x30) 
+L3GD20_HPFCF_0                       = const(0x00)
+L3GD20_HPFCF_1                       = const(0x01)
+L3GD20_HPFCF_2                       = const(0x02)
+L3GD20_HPFCF_3                       = const(0x03)
+L3GD20_HPFCF_4                       = const(0x04)
+L3GD20_HPFCF_5                       = const(0x05)
+L3GD20_HPFCF_6                       = const(0x06)
+L3GD20_HPFCF_7                       = const(0x07)
+L3GD20_HPFCF_8                       = const(0x08)
+L3GD20_HPFCF_9                       = const(0x09)
```

Comment:
I think you can remove those constants that are not needed.  Users can add them if they need them.